src/application/components/Layouts/SettingsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dispatch, Fragment, MutableRefObject, SetStateAction } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+
+declare const VERSION: string;
+
+export function Modal({
+  open,
+  setOpen,
+  cancelButtonRef,
+}: {
+  open: boolean;
+  cancelButtonRef: MutableRefObject<null>;
+  setOpen: Dispatch<SetStateAction<boolean>>;
+}) {
+  return (
+    <Transition.Root show={open} as={Fragment}>
+      <Dialog
+        as="div"
+        className="relative z-10"
+        initialFocus={cancelButtonRef}
```

Comment:
I don't see this `cancelButtonRef` added to any other element. From what I see in the [docs](https://headlessui.com/react/dialog#managing-initial-focus) this should be attached to some kind of element (likely a `<button />`). Do you need this at all? Perhaps leftover from a refactoring?

Reply:
Addressed in https://github.com/apollographql/apollo-client-devtools/pull/1128/commits/41bdf86d24adcb2decbd996401584e8c7914dc99